web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSTypedOM.kt

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2020-2021 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+@file:Suppress("UNUSED")
+package androidx.compose.web.css
+
+import kotlin.js.JsName
+
+expect interface CSSStyleValue
+
+open class CSSStyleValueKT : CSSStyleValue {
+    companion object {
+        fun parse(property: String, cssText: String): CSSStyleValue = TODO()
```

Comment:
is TODO() left intentionally?

Reply:
yes, it isn't used yet, and i think that maybe it should be removed from interfaces now